src/Accelerators/NNPA/Transform/ZHigh/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+add_subdirectory(Stickify)
```

Comment:
We removed stickify folder, so this is not necessary. Also we should remove everything about `ZHighConstPropagation` which depends on stickify.